app/src/main/java/org/jellyfin/mobile/car/MediaService.kt

```diff
@@ -0,0 +1,596 @@
+package org.jellyfin.mobile.car
```

Comment:
Shouldn't call this "car" since the media library / mediabrowser feature is not specifically for cars.